tools/biolatency.py

```diff
@@ -36,6 +37,8 @@
     help="millisecond histogram")
 parser.add_argument("-D", "--disks", action="store_true",
     help="print a histogram per disk device")
+parser.add_argument("-s", "--strip", action="store_true",
+    help="strip leading zero on output")
```

Comment:
I don't think this makes a lot of sense as an additional option for any tool. This is a decision the tool author should make. And in fact, I don't see why we wouldn't want this behavior by default -- so probably in most cases, the choice should be to strip leading zeros. @brendangregg